searx/engines/duckduckgo_extra.py

```diff
@@ -108,7 +109,7 @@ def _video_result(result):
         'title': result['title'],
         'content': result['description'],
         'thumbnail': result['images'].get('small') or result['images'].get('medium'),
-        'iframe_src': result['embed_url'],
+        'iframe_src': _get_iframe_src(result['content']),
```

Comment:
Is it really needed to call the function when the `result['embed_url']` already holds the url? .. Or is `result['embed_url']` sometimes failed?

Reply:
May I oversee something, but here we are in the ddg-video results and when parsing the result from ddg the `result['embed_url']` is already a *embedable* URL .. while `get_embeded_stream_url` was made to *"Converts a standard video URL into its embed format. Supported services include Youtube, Facebook, Instagram, TikTok, and Dailymotion."* .. and this is not needed for ddg-video results since they already serve a embedable URL / or I'm wrong?

Reply:
> So, if i search !ddv cats site:tiktok.com the result['embed_url'] does not work.

It was not clear to me, that we fix an issue in the ddg-video engine .. thanks :+1: